dspace/config/local.cfg.EXAMPLE

```diff
@@ -85,18 +85,41 @@ db.password = dspace
 #      so this may be set to ${db.username} in most scenarios.
 db.schema = public
 
-## Connection pool parameters
+## Database Connection pool parameters
+# DSpace wraps around the Apache Commons DBCP2 component, to read about its configuration
+# see: https://commons.apache.org/proper/commons-dbcp/configuration.html
 
-# Maximum number of DB connections in pool (default = 30)
-#db.maxconnections = 30
+# Maximum number of active DB connections in pool (-1 = unlimited)
+# (default = 30)
+db.maxconnections = 30
```

Comment:
Entries in `local.cfg.EXAMPLE` should always be commented out....as these are already the default settings in `dspace.cfg`.